README.md

```diff
@@ -1 +1,162 @@
+
+**# Working-with-APIs ** is a website that makes API calls
+
+## 🛠 Built With <a name="HTML"></a>
+
+### 🖥️ Tech Stack <a name="CSS"></a>
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- API
+
+### Key Features <a name="Mobile Version of Portfolio"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Three page web application**
+- **Use two API to display, store and update data**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+Please note that the live demo link is not yet active
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Go to this repsitory.
+- Click on Code
+- Clone the repository using your https link
+- This will gives you access to all files
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+
+- git clone git@github.com:Yemisirach/JavaScript-capstone-project.git
+
+- Once done, open the index.html file in live server
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+Because this project uses webpack as a bundler, to run the project, execute the following command:
+
+<code>npm run start</code>
+
+- OR
+
+<code>npm start</code>
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "\*_/_.{css,scss}" in your terminal
+- npx eslint . in your terminal
+
+### Deployment
+
+You can deploy this project using:
+
+A web browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Yemisrach**
+
+- GitHub: [@githubhandle](https://github.com/Yemisirach)
+
+👤 **Mihretab**
+
+- GitHub: [@githubhandle](https://github.com/damenam)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Making API Calls**
+
+## 🔭 Watch Video Explanation here <a name="video-demo"></a>
+
+- **Watch explanation of the features** [here]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Yemisirach/JavaScript-capstone-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+- If you like this project follow my github.
+- Give me star
+- Say hi on twitter
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- The MIT link provided is broken 😢, kindly provide a valid license link 👍🏽.

Reply:
The License link is fixed